src/POM/HomePage.js

```diff
@@ -0,0 +1,27 @@
+const {By} = require('selenium-webdriver');
+require('dotenv').config();
+
+class HomePage {
+	constructor(driver){
+		this.driver = driver;
+	}
+
+	async mainPage(){
+		await this.driver.get(process.env.HOMEPAGE_URL_LOCAL);
```

Comment:
This should have the main link in the env variables and then the view hardcoded after. For example:

process.env.APPLICATION_URL + "/login"
process.env.APPLICATION_URL + "/job-roles"

Reply:
done